### ISSUE TITLE
Container Image Build Fails - quarkus-container-image-jib

### ISSUE THREAD

**Issue by PStoner3** [quarkusio/quarkus#26193]
### Describe the bug

There appears to be an issue within the quarkus-container-image-jib extension while processing a container image build. The error seems to be focused around [line 233 in JibProcessor.java](https://github.com/quarkusio/quarkus/blob/e09da15f47c09b20082d7e9bc46216df69560c39/extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java#L233)

It appears the call to `jibContainerBuilder.containerize(containerizer);` is trying to make a connection to `registry.access.redhat.com` in an attempt to pull the `ubi8/openjdk-11-runtime:1.11` image. This call fails repeatedly with the following information:

> [WARNING] [io.quarkus.container.image.jib.deployment.JibProcessor] GET https://registry.access.redhat.com/v2/ubi8/openjdk-11-runtime/manifests/sha256:64acf3403b5c2c85f7a28f326c63f1312b568db059c66d90b34e3c59fde3a74b failed and will be retried

The build process ultimately fails with the following error/stacktrace

>[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:2.7.1.Final:build (default) on project location-lookup: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR]         [error]: Build step io.quarkus.container.image.jib.deployment.JibProcessor#buildFromJar threw an exception: java.lang.RuntimeException: Unable to create container image
[ERROR]         at io.quarkus.container.image.jib.deployment.JibProcessor.containerize(JibProcessor.java:234)
[ERROR]         at io.quarkus.container.image.jib.deployment.JibProcessor.buildFromJar(JibProcessor.java:161)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR]         at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[ERROR]         at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR]         at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[ERROR]         at io.quarkus.deployment.ExtensionLoader$2.execute(ExtensionLoader.java:882)
[ERROR]         at io.quarkus.builder.BuildContext.run(BuildContext.java:277)
[ERROR]         at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
[ERROR]         at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
[ERROR]         at java.base/java.lang.Thread.run(Thread.java:834)
[ERROR]         at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] Caused by: com.google.cloud.tools.jib.api.InsecureRegistryException: Failed to verify the server at https://registry.access.redhat.com/v2/ubi8/openjdk-11-runtime/manifests/sha256:64acf3403b5c2c85f7a28f326c63f1312b568db059c66d90b34e3c59fde3a74b because only secure connections are allowed.
[ERROR]         at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:179)
[ERROR]         at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:114)
[ERROR]         at com.google.cloud.tools.jib.registry.RegistryClient.callRegistryEndpoint(RegistryClient.java:625)
[ERROR]         at com.google.cloud.tools.jib.registry.RegistryClient.pullManifest(RegistryClient.java:436)
[ERROR]         at com.google.cloud.tools.jib.registry.RegistryClient.pullManifest(RegistryClient.java:441)
[ERROR]         at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.pullBaseImages(PullBaseImageStep.java:334)
[ERROR]         at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:157)
[ERROR]         at com.google.cloud.tools.jib.builder.steps.PullBaseImageStep.call(PullBaseImageStep.java:69)
[ERROR]         at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
[ERROR]         at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:69)
[ERROR]         at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
[ERROR]         at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[ERROR]         at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[ERROR]         at java.base/java.lang.Thread.run(Thread.java:834)
[ERROR] Caused by: javax.net.ssl.SSLException: Received fatal alert: internal_error
[ERROR]         at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:129)
[ERROR]         at java.base/sun.security.ssl.Alert.createSSLException(Alert.java:117)
[ERROR]         at java.base/sun.security.ssl.TransportContext.fatal(TransportContext.java:308)
[ERROR]         at java.base/sun.security.ssl.Alert$AlertConsumer.consume(Alert.java:279)
[ERROR]         at java.base/sun.security.ssl.TransportContext.dispatch(TransportContext.java:181)
[ERROR]         at java.base/sun.security.ssl.SSLTransport.decode(SSLTransport.java:164)
[ERROR]         at java.base/sun.security.ssl.SSLSocketImpl.decode(SSLSocketImpl.java:1152)
[ERROR]         at java.base/sun.security.ssl.SSLSocketImpl.readHandshakeRecord(SSLSocketImpl.java:1063)
[ERROR]         at java.base/sun.security.ssl.SSLSocketImpl.startHandshake(SSLSocketImpl.java:402)
[ERROR]         at org.apache.http.conn.ssl.SSLConnectionSocketFactory.createLayeredSocket(SSLConnectionSocketFactory.java:436)
[ERROR]         at org.apache.http.conn.ssl.SSLConnectionSocketFactory.connectSocket(SSLConnectionSocketFactory.java:384)
[ERROR]         at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:142)
[ERROR]         at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:376)
[ERROR]         at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:393)
[ERROR]         at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236)
[ERROR]         at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:186)
[ERROR]         at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
[ERROR]         at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
[ERROR]         at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:108)
[ERROR]         at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
[ERROR]         at com.google.api.client.http.apache.v2.ApacheHttpRequest.execute(ApacheHttpRequest.java:73)
[ERROR]         at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1012)
[ERROR]         at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:349)
[ERROR]         at com.google.cloud.tools.jib.http.FailoverHttpClient.call(FailoverHttpClient.java:266)
[ERROR]         at com.google.cloud.tools.jib.registry.RegistryEndpointCaller.call(RegistryEndpointCaller.java:138)
[ERROR]         ... 13 more
[ERROR] -> [Help 1]

Complete debug/trace output of this step is attached

I am behind a corporate firewall, so I disconnected from VPN and tried again with the same results. I recently moved to a different state and internet provider so I thought maybe my new provider may have been blocking somehow. I ran `docker pull registry.access.redhat.com/ubi8/openjdk-11-runtime@sha256:64acf3403b5c2c85f7a28f326c63f1312b568db059c66d90b34e3c59fde3a74b` with the following results:

```
registry.access.redhat.com/ubi8/openjdk-11-runtime@sha256:64acf3403b5c2c85f7a28f326c63f1312b568db059c66d90b34e3c59fde3a74b: Pulling from ubi8/openjdk-11-runtime
a9e23b64ace0: Already exists
38b71301a1d9: Already exists
21e3f6d4169e: Pulling fs layer
21e3f6d4169e: Verifying Checksum
21e3f6d4169e: Download complete
21e3f6d4169e: Pull complete
Digest: sha256:64acf3403b5c2c85f7a28f326c63f1312b568db059c66d90b34e3c59fde3a74b
Status: Downloaded newer image for registry.access.redhat.com/ubi8/openjdk-11-runtime@sha256:64acf3403b5c2c85f7a28f326c63f1312b568db059c66d90b34e3c59fde3a74b
registry.access.redhat.com/ubi8/openjdk-11-runtime@sha256:64acf3403b5c2c85f7a28f326c63f1312b568db059c66d90b34e3c59fde3a74b
```

I then ran a wireshark trace while running the build and found something very interesting. During the Jib build process, there are calls made from `host.docker.internal` to (at least in my case) `e40408.d.akamaiedge.net (23.48.11.106)` I'm assuming this is the pull of the image in question which do coincide with the "GET ... will be retried" messages. `host.docker.internal` sends a "Client Hello" packet. After an acknowledgement, `e40408.d.akamaiedge.net` returns a message "Alert (Level: Fatal, Description: Internal Error)". A sample of these packets have been dumped and attached

System Stats:
Maven:
```
$ mvn --version
Apache Maven 3.8.4 (9b656c72d54e5bacbed989b64718c159fe39b537)
Maven home: F:\development\Apache\Maven\Latest
Java version: 11, vendor: Oracle Corporation, runtime: C:\Users\aa99988\development\languages\Java\openJDK_11
Default locale: en_US, platform encoding: Cp1252
OS name: "windows 10", version: "10.0", arch: "amd64", family: "windows"
```

Java: 
```
$ java --version
openjdk 11 2018-09-25
OpenJDK Runtime Environment 18.9 (build 11+28)
OpenJDK 64-Bit Server VM 18.9 (build 11+28, mixed mode)
```

OS: Microsoft Windows 10 1809 210q - 20200218 [Version 10.0.19041.1415]

GIt for Windows: 
```
$ git --version
git version 2.36.1.windows.1
```

Git-bash:
```
$ bash --version
GNU bash, version 4.4.23(1)-release (x86_64-pc-msys)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
```

Quarkus version:
```
$ quarkus --version
2.9.2.Final
```
[jibprocessor_fail.txt](https://github.com/quarkusio/quarkus/files/8921127/jibprocessor_fail.txt)
[jib_fail_alert_fatal.zip](https://github.com/quarkusio/quarkus/files/8921317/jib_fail_alert_fatal.zip)

### Expected behavior

The build would complete and a container image of the application would be available for use.

### Actual behavior

Build fails as described above

### How to Reproduce?

Using quarkus-quickstarts:getting-started. I added the following properties to `application.properties`

'''
quarkus.native.container-runtime=docker
quarkus.native.container-build=true

quarkus.container-image.build=true
quarkus.container-image.name=getting-started-test
'''

A zip file of this minimally reproducible project is attached
[getting-started.zip](https://github.com/quarkusio/quarkus/files/8921673/getting-started.zip)

**Comment by PStoner3** [comment]
After further investigation, I realized the issue was indeed on my side. I also found the setting quarkus.container-image.insecure, set it to true and my build finally finished successfully.

It would appear that certain credentials were not set/provided on myside. Though I'm puzzled as to why this would have worked before now.

at any rate, I have an acceptable solution for now. I consider this matter closed
